core/scripts/testing/themes/dark.css

```diff
@@ -53,19 +53,19 @@
   --ion-color-dark-shade: #d7d8da;
   --ion-color-dark-tint: #f5f6f9;
 
-  --ion-color-medium: #989aa2;
-  --ion-color-medium-rgb: 152, 154, 162;
+  --ion-color-medium: #86919c;
```

Comment:
Medium and light are extremely close, even though they pull from the carbon vs. gray design tokens. 

I'm open to suggestions here on how we can strike a balance of color contrast with a wider color separation between light and medium.

Reply:
I actually think the `#959595` color you used for light looks closer to what was there. Does that one work?